src/component/marker/MarkAreaView.ts

```diff
@@ -123,6 +123,20 @@ function markAreaFilter(coordSys: CoordinateSystem, item: MarkAreaMergedItemOpti
         ) {
             return true;
         }
+        /*Another zone filter is applied, which might not be necessary.
+        Directly returning true means displaying markArea component permanently,
```

Comment:
We prefer using inline comment // here.

Also this comment seems to be outdated after new commits?

Reply:
For this one, I mean we are using a function `zonefilter` here, but actually returning true can achieve almost the same effect as using this function. The reason is that, when returning true, `markArea` is always shown and will not be clipped however the grid changes, which solves the problem. When the coordinates does not include the `markArea`, it's automatically not shown. So in my opinion  a function to check if the coordinates intersect with `markArea` is unnecessary. Of course, keeping the `markArea` rendered may be problematic when there are huge amount of them, but in most cases one or two `markArea`s would hardly affect performance.

Reply:
Got it. Thanks for the detailed explanation. Perhaps we can tweak the description a bit to make it more clear. For example:

Returning true directly may also do the work. But filtering ahead will avoid creating too much unnecessary elements when there are huge amount of elements.

Reply:
Good point. Comment updated.